packages/core/types/src/modules/entity-service/params/attributes/index.ts

```diff
@@ -49,23 +23,23 @@ export type ScalarValues = GetValue<
   | Attribute.Integer
   | Attribute.Blocks
   | Attribute.JSON
-  // /!\  Password attributes are NOT filterable and should NOT be part of this union type.
-  //      The member below has been commented on purpose to avoid adding it back without noticing.
-  // | Attribute.Password
   | Attribute.RichText
   | Attribute.String
   | Attribute.Text
   | Attribute.Time
   | Attribute.Timestamp
   | Attribute.UID<Common.UID.Schema>
+  // /!\  Password attributes are NOT filterable and should NOT be part of this union type.
+  //      The member below has been commented on purpose to avoid adding it back without noticing.
+  // | Attribute.Password
 >;
 
 /**
  * Attribute.GetValues override with extended values
  */
 export type GetValues<TSchemaUID extends Common.UID.Schema> = {
   id?: ID;
-} & OmitRelationWithoutTarget<
+} & OmitRelationsWithoutTarget<
```

Comment:
I'm wondering if renaming OmitRelationsWithoutTarget (as well as moving the export path of it, along with ID and literals) would be considered a breaking change, since we do export it in @strapi/strapi, even though it's not documented.

Could we get around that by also continuing to export them the same way from this file, with a note to remove the dual-export in v5?

Reply:
Root namespaces are unchanged, which I think should be the way the "types breaking changes" should be looked at.

We should advise people to use root-level namespace imports (`@strapi/strapi`) instead of deep ones (`@strapi/strapi/core/entity-service/...`), which -in my opinion- correspond to the public type API

Hence, since the splitting offers more readability from a maintainer perspective (at least I believe so), I would keep it personally like this. What do you think?

Reply:
Ok, I agree, I think as long as we keep the namespaces the same people shouldn't be using the deep references.